students/Cracolici_Jon/lesson04/JonCracolici_Kata.py

```diff
@@ -0,0 +1,76 @@
+#Jon Cracolici
+#UW-Python Cert
+#Lesson 4 Kata Exercise
+
+#This lesson focuses on building a new text document in the style of a 
+#provided text document using the dictionary data structure.
+
+#I will need 4 basic functions: a way to import text, a way to parse it,
+#a way to build the Kata dict, and a way to use the Kata dict to create 
+#and write a new file. Lets go!
+
+import random as rand
+def import_parse_text(file):
+    """This function takes a text file, opens it and parses it into a list of lower case words."""
+    #File import portion
+    localfile = open(file, 'r')
```

Comment:
In Python you'll want all of these to be snake_case just like your function names: local_file

Reply:
prefer to use `with` when working with files